src/core/include/openvino/op/convert.hpp

```diff
@@ -41,7 +41,7 @@ class OPENVINO_API Convert : public Op {
     }
 
     OPENVINO_SUPPRESS_DEPRECATED_START
-    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
+    bool evaluate(ov::TensorVector& output_values, const ov::TensorVector& input_values) const override;
```

Comment:
I suppose it should not be wrapped with `OPENVINO_SUPPRESS_DEPRECATED_START` now

Reply:
removed